xml2json/xml2json-tests.ts

```diff
@@ -0,0 +1,29 @@
+import * as parser from 'xml2json'
+
+var xml = "<foo attr=\"value\">bar</foo>";
+console.log("input -> %s", xml)
+
+// xml to json 
+var json = parser.toJson(xml);
+console.log("to json -> %s", json);
```

Comment:
`console.log` doesn't check the type of its arguments. Try e.g. `var json: string = ...`.